test/runtests.jl

```diff
@@ -38,6 +38,7 @@ if GROUP == "All" || GROUP == "InterfaceI" || GROUP == "Interface"
   @time @safetestset "Composite Interpolation Tests" begin include("interface/composite_interpolation.jl") end
   @time @safetestset "Export tests" begin include("interface/export_tests.jl") end
   @time @safetestset "Type Handling Tests" begin include("interface/type_handling.jl") end
+  @time @safetestset "Controller Tests" begin include("interface/controllers.jl") end
```

Comment:
oh no, was that missing before? Does this have a test case for this?

Reply:
I added this test case here (but noticed that I forgot to push it in the first commit - fixed in e886266e). This file contains a new test case for this specific case.

Reply:
Ahh okay. Slap a few other controllers in there and see if they have a similar issue with 0 EEst?

Reply:
I did that - everything looks good so far